### PR TITLE
Create Nida8930.md

### PR DESCRIPTION
**Pull request by Nida8930**
https://raw.githack.com/Nida8930/Nida8930.github.io/initial-branch/#!pages/vi/profiles/Nida8930.md

### CODE REVIEW

File: pages/vi/profiles/Nida8930.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Nicholas Davis
+![alt-text](https://img1.southernliving.timeinc.net/sites/default/files/styles/4_3_horizontal_-_1200x900/public/1542301326/GettyImages-1033240660.jpg?itok=6Sh4z7Za "Tulsa, Ok")
+
+About Me
+----
+Hello Everyone,
+
```
Comment (CalebProvost) [inline pages/vi/profiles/Nida8930.md:7]: Everything is good and meets the minimum requirements, but lets add in the timezone and system being used wherever you see fit:

```
Then, in the new Markdown file, include the following information:
   * your name, location/time zone, and OS (including version)
   * a short description of yourself (so your fellow virtual interns know who you are)
```

**Review by CalebProvost — CHANGES_REQUESTED** [review]

File: pages/vi/profiles/Nida8930.md (hunk 1 above)
Comment (Nida8930) [reply pages/vi/profiles/Nida8930.md:7]: Hey Caleb,
Unless something isn't loading correctly and I'm seeing something different my time zone and OS info is located at the bottom of the page under the general info section. Let me know if there is an issue that I'm not seeing. Thanks!

Comment (CalebProvost) [reply pages/vi/profiles/Nida8930.md:7]: Nope. You are absolutely correct. I'm not sure why it didn't register in my brain. My eyes just glazed right over that.
Sorry about that.

**Review by CalebProvost — APPROVED** [review]

File: pages/vi/profiles/Nida8930.md (hunk 1 above)
Comment (Nida8930) [reply pages/vi/profiles/Nida8930.md:7]: No worries, it's early, glad everything looks good.

**Review by pattanawadee88 — APPROVED** [review]

File: pages/vi/profiles/Nida8930.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+General Info
+----
+- From Tulsa, Oklahoma (Central Standard Time)
+- Working on MacOS Sierra and Ubuntu
+
+```python
+def Hello():
+  return "Hello, Open Learning Exchange!"
```
Comment (dogi) [inline pages/vi/profiles/Nida8930.md:18]: hello @Nida8930 ;)

**Pull request merged**